ccmlib/node.py

```diff
@@ -1491,8 +1498,10 @@ def _find_pid_on_windows(self):
 
     def _get_directories(self):
         dirs = []
-        for i in ['data', 'commitlogs', 'saved_caches', 'logs', 'conf', 'bin', os.path.join('data', 'hints')]:
+        for i in ['commitlogs', 'saved_caches', 'logs', 'conf', 'bin', 'hints']:
```

Comment:
Is this going to correctly store hints for versions where data/hints is the default hints directory?